pettingzoo/classic/tictactoe/tictactoe.py

```diff
@@ -153,88 +155,70 @@ def __init__(
         if self.render_mode == "human":
             self.clock = pygame.time.Clock()
 
-    # Key
```

Comment:
This comment seems useful, is it no longer accurate?

Reply:
It was not correct. The observations are not a list of lists and they are not the values shown. They are either 1 or 0. I didn't change that, it was like before.

The board values themselves are what is shown and that is documented in the board class.